### ISSUE TITLE
Calypsofy - Feedback UI sidebar is broken on RTL interfaces

### ISSUE THREAD

**Issue by jjchrisdiehl** [Automattic/wp-calypso#46361]
#### Steps to reproduce

1. Set Calypso interface language to any RTL (I used Hebrew in this instance)
2. Navigate to My Site > Site > Feedback to open the calypsofied Feedback page

#### What I expected

English interface

![image](https://user-images.githubusercontent.com/16580129/95759823-39c8c580-0c78-11eb-9214-fb51b35c6483.png)

Or rather I suppose more like this (English image flipped)

![image](https://user-images.githubusercontent.com/16580129/95760197-b8256780-0c78-11eb-8714-2f5a3aaf41fd.png)

#### What happened instead
Right aligned Calypso sidebar obscures content, doesn't provide enough room. Also, the Switch Sites/Back button stays left aligned.

![image](https://user-images.githubusercontent.com/16580129/95759609-eeaeb280-0c77-11eb-8cef-f4b015e5e05e.png)

#### Browser / OS version
Microsoft Edge Chromium
Version 86.0.622.38 (Official build) (64-bit)

#### Temp workaround
Customer can load that page with the calypsofy flag set to 0:

`https://domain.wordpress.com/wp-admin/edit.php?post_type=feedback&calypsoify=0`

**Comment by yansern** [comment]
Some findings:
- the url to test is your_site_dot_wordpress_dot_com/wp-admin/edit.php?post_type=feedback&calypsoify=1`
- the element `#calypso-sidebar-header` should have a `right: 0` instead of `left: 0`
- the left and right margins on `#wpcontent` wasn't flipped as well
- the relevant file is `wp-content/admin-plugins/calypsoify/style.css` on wpcom
- there seems to be no RTL equivalent file generated for the style.css file above

Next actions:
- Find out why the RTL equivalent wasn't generated.

**Comment by jeyip** [comment]
I can reliably reproduce this bug. I agree with the priority label of `high` because, as mentioned above, it obscures content, arguably making the site unusable. Moving this into the create focus maintenance backlog now.

**Comment by jeyip** [comment]
Relevant excerpt from PCYsg-1zS-p2:

"if your CSS file is name style.css it will be ignored, so you will need to create the RTL version manually." Because the relevant styles are generated from `wp-content/admin-plugins/calypsoify/style.css`, it sounds like this is expected behavior. The guide recommends that we manually create the rtl file ourselves.

**Comment by jeyip** [comment]
D52490-code

**Comment by jeyip** [comment]
I verified that this issue is now fixed on my dotcom websites
